app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -91,18 +110,19 @@ public void updateFromItem(final InfoItem infoItem,
 
         itemThumbnailView.setOnClickListener(view -> openCommentAuthor(item));
 
+        try {
+            streamService = NewPipe.getService(item.getServiceId());
+        } catch (final ExtractionException e) {
+            // should never happen
+            Log.w(TAG, "Cannot obtain service from comment service id, defaulting to YouTube", e);
```

Comment:
Shouldn't we display an error snackbar? Otherwise, we won't notice the bug itself, but might get complains about wrong behavior.

Reply:
Right, done